clang/lib/Driver/SanitizerArgs.cpp

```diff
@@ -68,6 +68,8 @@ static const SanitizerMask TrappingSupported =
     SanitizerKind::ImplicitConversion | SanitizerKind::Nullability |
     SanitizerKind::LocalBounds | SanitizerKind::CFI |
     SanitizerKind::FloatDivideByZero | SanitizerKind::ObjCCast;
+static const SanitizerMask MergeDefault = SanitizerKind::Undefined;
+static const SanitizerMask MergeSupported = SanitizerKind::Undefined;
```

Comment:
Having that this is advisory feature, maybe we don't need to be strict on support checking,
so no need to MergeSupported

Reply:
Removed